docs/usage.rst

```diff
@@ -0,0 +1,18 @@
+Usage
+=====
+
+This page is a work in progress and details the usage of Astoria.
+
+Metadata USBs
+-------------
+
+Metadata USBs contain overrides for the metadata.
+
+The override data should be stored in a file called ``astoria.json`` in JSON format.
+
+There should ideally not be any other files on the Metadata USB. If a ``robot.zip`` is placed on the USB, it will cause interference.
```

Comment:
probably good to document exactly what happens in this case, though doesn't have to be done in this pr

Reply:
I think changing it in this PR might cause some painful merge conflicts with #20 so I'll raise an issue instead.